curation_concerns-models/lib/generators/curation_concerns/models/fulltext_generator.rb

```diff
@@ -17,7 +17,7 @@ def banner
   # Copy CurationConcerns's solrconfig into the dir from which the jetty:config task pulls
   # CurationConcerns's solrconfig includes full-text extraction
   def copy_solr_config
-    src = File.join(__FILE__, '..', '..', '..', '..', '..', '..', 'solr_conf', 'conf', 'solrconfig.xml')
+    src = File.join(__FILE__, '..', 'templates', 'config', 'solrconfig.xml')
     copy_file File.expand_path(src), 'solr_conf/conf/solrconfig.xml', force: true
```

Comment:
No need to do this expand path anymore, just `copy_file 'config/solrconfig.xml', 'solr_conf/conf/solrconfig.xml', force: true`